### PR TITLE
Add Detroit styles, logos, and text to the partner portal

### PR DESCRIPTION
**Pull request by elenm**
CityOfDetroit/affordable-housing-app/issues/77

- Changes the color scheme of the partner portal to match Detroit's style guideline
- Adds Detroit logos to the partner portal 
- Adds a Detroit specific footer and header text

after: sign in
before: sign in
after: listings page
before: listings page

after: sign in mobile
before: sign in mobile
after: listings mobile
before: listings mobile

### CODE REVIEW

File: sites/partners/layouts/index.tsx [hunk 1]
```diff
@@ -40,7 +40,19 @@ const Layout = (props) => {
         <main>{props.children}</main>
 
         <SiteFooter>
-          <FooterNav copyright="© 2020 • All Rights Reserved">
+          <FooterSection>
```
Comment (avaleske) [inline sites/partners/layouts/index.tsx:43]: Would it make sense to pull this out into a shared component? It seems (at first glance) similar to the footer on the public site.

File: sites/partners/styles/overrides.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// The header logo should allow for a larger maximum height than the default
+// logo.
+.navbar-brand .logo .logo__image {
```
Comment (avaleske) [inline sites/partners/styles/overrides.scss:3]: I was about to suggest using Tailwind's min-height classes so we don't have some styles that are tailwind and some that aren't, but it seems like we'd have to add custom min-height anyway:
https://tailwindcss.com/docs/min-height
They have some for max-height though:
https://tailwindcss.com/docs/max-height

So up to you if you want to bother trying to get it to work that way

**Review by avaleske — APPROVED** [review]
Thanks for doing this!

File: sites/partners/layouts/index.tsx (hunk 1 above)
Comment (elenm) [reply sites/partners/layouts/index.tsx:43]: We can make them a shared component, but I'm not yet sure if the footers will end up being different between the partner portal and applicant site. This is just my initial guess at adding a footer here. I will keep it as is until I'm sure they should be a shared component.

File: sites/partners/styles/overrides.scss (hunk 2 above)
Comment (elenm) [reply sites/partners/styles/overrides.scss:3]: I'll keep this as is right now, but will reach out to you directly to make sure I'm understanding your suggestion.

**Pull request merged**